modules/javafx.graphics/src/main/native-glass/mac/GlassViewDelegate.m

```diff
@@ -973,7 +973,8 @@ - (void)startDrag:(NSDragOperation)operation withItems:(NSArray<NSDraggingItem*>
         if (image == nil && [pbItemTypes containsObject:NSPasteboardTypeFileURL])
         {
             // create an image with contents of URL
-            image = [[NSImage alloc] initByReferencingFile:[[NSString alloc] initWithData:[pbItem dataForType:NSPasteboardTypeFileURL] encoding:NSUTF8StringEncoding]];
+            NSString *fileURL = [[[NSString alloc] autorelease] initWithData:[pbItem dataForType:NSPasteboardTypeFileURL] encoding:NSUTF8StringEncoding];
```

Comment:
Minor: The generally accepted pattern is to call `autorelease` after the `init`, so as to not split `alloc` and `init`, so:

```
NSString *fileURL = [[[NSString alloc] initWithData:[pbItem dataForType:NSPasteboardTypeFileURL]
    encoding:NSUTF8StringEncoding] autorelease];
```